cli/tests/e2e-pro/test_ci.py

```diff
@@ -1297,6 +1335,16 @@ def test_shallow_wrong_merge_base(
     subprocess.run(["git", "fetch", "origin", "--depth", "1", "bar:bar"])
     subprocess.run(["git", "checkout", "bar"], check=True, capture_output=True)
 
+    start_scan_mock = start_scan_mock_maker(
+        env.get("SEMGREP_URL", "https://semgrep.dev")
```

Comment:
Do these need to have env.get aka is SEMGREP_URL ever set on this test?

Reply:
Not at this point, I'll remove it.